.licenserc.yaml

```diff
@@ -0,0 +1,36 @@
+header:
```

Comment:
I wouldn't include this file unless we need some extra exclusions, which shouldn't be the case at this point. The default is https://github.com/canonical/operator-workflows/blob/main/.github/files/.licenserc.yaml

Reply:
Thanks!